tasks/pkg-debian.yml

```
@@ -116,6 +116,12 @@
     update_cache: true
   when: (datadog_apt_repo | length > 0) and (not ansible_check_mode)
 
+- name: Remove /opt/datadog-agent/python-scripts/__pycache__
+  ansible.builtin.file:
+    path: /opt/datadog-agent/python-scripts/__pycache__
+    state: absent
+  when: not agent_datadog_skip_install
```

```suggestion
  when: not agent_datadog_skip_install
  ignore_errors: true
```
Let's add a safeguard to not block the whole playbook if for some reason this task fails